mcs/class/referencesource/System.Data.SqlXml/System/Xml/Xsl/Xslt/XslAstAnalyzer.cs

```diff
@@ -1287,7 +1287,7 @@ internal sealed class XslAstRewriter {
                         continue;
                     }
 
-                    // Need to create a new QilName (can't reuse the one from the variable, eventhough it's exactly the same)
+                    // Need to create a new QilName (can't reuse the one from the variable, even though it's exactly the same)
```

Comment:
Please revert changes to files in mcs/class/referencesource, we imported that code from https://github.com/Microsoft/referencesource and fixing typos is not worth it there.